fixup-mountpoints

```diff
@@ -420,23 +420,28 @@ case "$DEVICE" in
 
     "m7spr")
         sed -i \
+            -e 's block/platform/msm_sdcc.1/by-name/adsp mmcblk0p16 ' \
             -e 's block/platform/msm_sdcc.1/by-name/boot mmcblk0p35 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/cache mmcblk0p38 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/misc mmcblk0p19 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/radio mmcblk0p33 ' \
             -e 's block/platform/msm_sdcc.1/by-name/recovery mmcblk0p36 ' \
             -e 's block/platform/msm_sdcc.1/by-name/system mmcblk0p37 ' \
-            -e 's block/platform/msm_sdcc.1/by-name/cache mmcblk0p38 ' \
             -e 's block/platform/msm_sdcc.1/by-name/userdata mmcblk0p39 ' \
             "$@"
         ;;
 
     "t6")
         sed -i \
+            -e 's block/platform/msm_sdcc.1/by-name/adsp mmcblk0p16 ' \
             -e 's block/platform/msm_sdcc.1/by-name/boot mmcblk0p37 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/cache mmcblk0p41 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/misc mmcblk0p21 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/radio mmcblk0p34 ' \
             -e 's block/platform/msm_sdcc.1/by-name/recovery mmcblk0p38 ' \
             -e 's block/platform/msm_sdcc.1/by-name/system mmcblk0p39 ' \
-            -e 's block/platform/msm_sdcc.1/by-name/modem_st1 mmcblk0p22 ' \
-            -e 's block/platform/msm_sdcc.1/by-name/modem_st2 mmcblk0p23 ' \
-            -e 's block/platform/msm_sdcc.1/by-name/cache mmcblk0p41 ' \
             -e 's block/platform/msm_sdcc.1/by-name/userdata mmcblk0p42 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/wcnss mmcblk0p18 ' \
```

Comment:
Could you please squash this commit into the main one (the main commit should be the only commit in this PR)

Reply:
Has been re-fixed.